debos/qemu-boot-efi.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+qemu-system-x86_64 \
+    -enable-kvm -cpu host -smp 2 -m 1G \
```

Comment:
Is there any benefit to increasing this to our minimum system requirements?

Reply:
I'm not sure I'll even keep this file, tbh, it just made testing easier early on